src/framework/standard/command.rs

```diff
@@ -113,10 +115,29 @@ impl Default for CommandGroup {
             owners_only: false,
             allowed_roles: Vec::new(),
             help: None,
+            checks: Vec::new(),
         }
     }
 }
 
+impl fmt::Debug for CommandGroup {
```

Comment:
Since `checks` is using a struct with `Debug` impl'd now, this can be replaced with `#[derive(Debug)]`.